tests/unit/taskmodules/test_transformer_token_classification.py

```diff
@@ -0,0 +1,1042 @@
+        targets.append(taskmodule.encode_target(task_encodings_without_targets[ctr][i]))
+    return targets
+
+
+def test_target(targets, taskmodule, config):
+    labels_tokens = []
+
+    # If config is empty
+    if config == {}:
+        assert len(targets) == 1
+        """
+        expected_input_tokens = ["[CLS]","mount","everest","is","the","highest","peak","in","the","world",".","[SEP]",]
+        """
+        expected_labels = [
+            ["<pad>", "B-head", "I-head", "O", "O", "O", "O", "O", "O", "O", "O", "<pad>"]
+        ]
+        labels_tokens.append(
+            [taskmodule.id_to_label[x] if x != -100 else "<pad>" for x in targets[0]]
+        )
+
+    # If config has the specified values (max_window=8, window_overlap=2)
+    elif config == {"max_window": 8, "window_overlap": 2}:
+        assert len(targets) == 4
+        """
+        expected_input_tokens = [
+            ["[CLS]","mount","everest","is","the","highest","peak","[SEP]"],
+            ["[CLS]","is","the","highest","peak","in","the","[SEP]"],
+            ["[CLS]","highest","peak","in","the","world",".","[SEP]"],
+            ["[CLS]", "in", "the", "world", ".", "[SEP]"],
+        ]
+        """
+        expected_labels = [
+            ["<pad>", "B-head", "I-head", "O", "O", "<pad>", "<pad>", "<pad>"],
+            ["<pad>", "<pad>", "<pad>", "O", "O", "<pad>", "<pad>", "<pad>"],
+            ["<pad>", "<pad>", "<pad>", "O", "O", "O", "O", "<pad>"],
+            ["<pad>", "<pad>", "<pad>", "O", "O", "<pad>"],
+        ]
+        for i in range(len(targets)):
+            labels_tokens.append(
+                [taskmodule.id_to_label[x] if x != -100 else "<pad>" for x in targets[i]]
+            )
+
+    # If config has the specified value (max_window=8)
+    elif config == {"max_window": 8}:
+        assert len(targets) == 2
+        """
+        expected_input_tokens = [
+            ["[CLS]","mount","everest","is","the","highest","peak","[SEP]"],
+            ["[CLS]", "in", "the", "world", ".", "[SEP]"]
+        ]
+        """
+        expected_labels = [
+            ["<pad>", "B-head", "I-head", "O", "O", "O", "O", "<pad>"],
+            ["<pad>", "O", "O", "O", "O", "<pad>"],
+        ]
+        for i in range(len(targets)):
+            labels_tokens.append(
+                [taskmodule.id_to_label[x] if x != -100 else "<pad>" for x in targets[i]]
+            )
+
+    # If config has the specified value (partition_annotation=sentences)
+    elif config == {"partition_annotation": "sentences"}:
+        assert len(targets) == 1
+        """
+        expected_input_tokens = ["[CLS]", "bob", "enjoys", "playing", "soccer", ".", "[SEP]"]
+        """
+        expected_labels = [["<pad>", "O", "O", "O", "O", "O", "<pad>"]]
+        labels_tokens.append(
+            [taskmodule.id_to_label[x] if x != -100 else "<pad>" for x in targets[0]]
+        )
+
+    else:
+        raise ValueError("unknown config: {config}")
```

Comment:
Sorry, that was my fault, this needs to be the following (note the `f`), otherwise the config is not shown, but just the string `{config}` (please also fix the other occurrences):

```suggestion
        raise ValueError(f"unknown config: {config}")
```